src/tools/device-information/device-information.vue

```diff
@@ -89,7 +89,7 @@ const sections = [
   .value {
     font-size: 20px;
     font-weight: 400;
-    line-height: 1;
+    margin-top: -5px;
```

Comment:
I not a huge fan of negative margin, I think we can just get ride of the `line-height: 1`

```suggestion
```